index.jsx

```diff
@@ -46,42 +47,21 @@ import { loadCommands } from './lib/actions/terminalActions';
 import { loadSettings } from './lib/actions/uiActions';
 
 const supportedBoards = ['PCA10090', 'PCA10064', 'PCA20035', 'THINGY91'];
-const platform = process.platform.slice(0, 3);
-
-/* eslint react/prop-types: 0 */
-
-// Prefer to use the serialport 8 property or fall back to the serialport 7 property
-const portPath = serialPort => serialPort.path || serialPort.comName;
 
 /**
- * Pick the serialport that should belong to the modem on PCA10090
- * @param {Array<device>} serialports array of device-lister serialport objects
+ * Pick the serialport first serial port as this list should
+ * now be ordered deterministically.
+ * @param {Array<device>} serialports array of device-lib serialport objects
  * @returns {object} the selected serialport object
  */
-function pickSerialPort(serialports) {
-    if (serialports.length === 1) {
-        // Just in case a PCA10064 is selected or macOS case when serialports are split
-        return serialports[0];
-    }
-    switch (platform) {
-        case 'win':
-            return serialports.find(s => (/MI_00/.test(s.pnpId)));
-        case 'lin':
-            return serialports.find(s => (/-if00$/.test(s.pnpId)));
-        case 'dar':
-            // this doesn't work, but with fixDevices() can't happen
-            return serialports.find(s => (/1$/.test(portPath(s))));
-        default:
-    }
-    return undefined;
-}
+const pickSerialPort = serialports => serialports[0];
```

Comment:
That the order is deterministic does not explain to me why the first one is the correct port. I suspect, that the first port should always be one with a certain criteria. Is it always the port to the modem terminal? 

Please also express this criteria in the function name (With that, an additional comment may not even be needed).

Reply:
With the new library its enough to select the port with a vcom-value of 0 to identify the serial command port, which I've decided to do inline now.